tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java

```diff
@@ -1,28 +1,5 @@
 package org.unicode.cldr.unittest;
 
-import com.google.common.base.Joiner;
```

Comment:
@srl295 I think the Eclipse formatter is fighting a battle with spotless. So have to figure out how to disable that.

Reply:
Also, the maven build is having trouble: Error:  Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:3.3.1:resources (default-resources) on project cldr-code: Execution default-resources of goal org.apache.maven.plugins:maven-resources-plugin:3.3.1:resources failed: Plugin org.apache.maven.plugins:maven-resources-plugin:3.3.1 or one of its dependencies could not be resolved: Failed to collect dependencies at org.apache.maven.plugins:maven-resources-plugin:jar:3.3.1 -> org.codehaus.plexus:plexus-interpolation:jar:1.26: Failed to read artifact descriptor for org.codehaus.plexus:plexus-interpolation:jar:1.26: Could not transfer artifact org.codehaus.plexus:plexus-interpolation:pom:1.26 from/to central (https://repo.maven.apache.org/maven2): transfer failed for https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-interpolation/1.26/plexus-interpolation-1.26.pom: Unknown host repo.maven.apache.org: Name or service not known -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :cldr-code
Error: Process completed with exit code 1.